DOCS/canonical-identity-platform-reference-login-ui-observability.md

```diff
@@ -2,6 +2,9 @@ The Identity Platform Login UI Operator of the IAM Platform provides integration
 
 In this reference we will provide a technical description of the alert rules, and the dashboards the Identity Platform Login UI operator passes to the components of the COS. We will also showcase the Metrics we use to provide observability features.
 
+> See more: [metrics-endpoint and grafana dashboard integrations](https://charmhub.io/identity-platform-login-ui/integrations)
```

Comment:
I think we can say it like on https://github.com/canonical/canonical-identity-platform-docs/blob/main/DOCS/canonical-identity-platform-how-to-integrate-with-oidc-compatible-charms.md?plain=1#L13
```suggestion
> See more: [Charmhub | Identity Platform Login Ui Operator > Integrations > `metrics-endpoint`,  `grafana-dashboard`](https://charmhub.io/identity-platform-login-ui-operator/integrations)
```

BTW the link is wrong, so we need to fix this even if we don't change the format